packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -155,6 +155,27 @@ function makeAugmentedSchema({
             },
         });
     });
+
+    // Generates the mutations for enums, which are reused
+    Array.from(enumTypes.values()).forEach((enumType) => {
```

Comment:
I would start moving these outside this large `makeAugmentedSchema` function as a separate step!